tests/e2e-tests/src/deployment-api.test.ts

```diff
@@ -48,17 +53,15 @@ describe('DeploymentApi', () => {
     );
 
     createdDeploymentId = runningDeployment.id;
-  }, 180000);
+  }, 200000);
 
   it('should modify the deployment to stop it', async () => {
-    const deploymentId = getDeploymentId(createdDeploymentId);
-
-    const modifyResponse = await DeploymentApi.deploymentModify(
-      deploymentId,
-      { targetStatus: 'STOPPED' },
-      { 'AI-Resource-Group': resourceGroup }
-    ).execute();
+    const deploymentId = await checkCreatedDeployment(
```

Comment:
@marikaner @shibeshduw  I think I am missing the point here, what would be the reason I would run modify deployment individually, without having run create deployment? Right now, if no deployment exists, this function throws an error anyways. 
Its the same behavior if I call the modify script, the `createdDeploymentId` is undefined in that case, and if no other deployments exist, we still throw.

Reply:
I think the best way to go about it would be to create a separate script to cleanup deployments like @marikaner suggested. We can make this a GitHub action that runs before the e2e tests and also be able to run it manually when testing locally. 
Only then would the current impl make sense. Because right now, it doesn't make sense like you said. The point of running the test individually was so that if something is wrong, the entire test flow doesn't have to be run every time. 

I will make this change today as I'm also not satisfied with the current way.

Reply:
Okay I made the above change. Now makes sense to run the tests either individually or all at once. The cleanup script is setup in github workflows so we don't have to check for test failures and the million different conditions that come with it. Let me know if this works.